components/cloze/actions/create-update-company/create-update-company.mjs

```diff
@@ -127,6 +135,10 @@ export default {
       additionalData,
     } = this;
 
+    if (!name && !domains) {
+      throw new ConfigurationError("**Company Name** or **Domains** are required.");
```

Comment:
I think this text could be tweaked, e.g.
```suggestion
      throw new ConfigurationError("Either **Company Name** or **Domains** are required.");
```
Since only one of them is required